ccd-definition/CaseTypeTab/CareSupervision/4_DraftOrders.json

```diff
@@ -1,4 +1,37 @@
 [
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "DraftOrdersTab",
+    "TabLabel": "Draft orders",
+    "TabDisplayOrder": 4,
+    "CaseFieldID": "draftSDOLabel",
+    "TabFieldDisplayOrder": 1,
+    "FieldShowCondition": "standardDirectionOrder.orderStatus!=\"SEALED\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "DraftOrdersTab",
+    "TabLabel": "Draft orders",
+    "TabDisplayOrder": 4,
+    "CaseFieldID": "standardDirectionOrder",
+    "TabFieldDisplayOrder": 2,
+    "FieldShowCondition": "standardDirectionOrder.orderStatus!=\"SEALED\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "DraftOrdersTab",
+    "TabLabel": "Draft orders",
+    "TabDisplayOrder": 4,
+    "CaseFieldID": "draftSDOLinkDraftOrdersTab",
+    "TabFieldDisplayOrder": 3,
```

Comment:
are the numbers correct, they are repeated in cmo part

Reply:
but, yeah, it was signed off this way

Reply:
CMO and SDO won't be there at the same time so it is okay for the numbers to be the same

Reply:
yeah, true for draft, but similar situation in orders